wow/views.py

```diff
@@ -39,6 +39,7 @@ def clean_addr_dict(addr):
         "bin": str(addr['bin']),
         "lastsaleamount": int_or_none(addr['lastsaleamount']),
         "registrationid": str(addr['registrationid']),
+        "allcontacts": json_or_none(addr['allcontacts'])
```

Comment:
Er wait... why is this coming out of the database as stringified JSON (rather than parsed JSON) again?  I remember this is what you pinged me about on Slack but I'm unclear on why it's happening.  Maybe document it in the code so that future readers aren't befuddled?

Reply:
Ooo I think it _might_ have something to do with this: https://code.djangoproject.com/ticket/31991

While we are executing raw SQL, we're doing it through Django as an intermediary, which might be the culprit.  Gah so annoying.

Reply:
Figured out a better fix, just filed #480!